app/views/foods/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Total number of items with sweet potatoes: <%= %></h2>
+<h2>Ten foods that contain sweet potatoes:</h2>
+<ul>
+  <% @foods.each do |food| %>
+  <li>
+      Code: <%= food.code %> |
+      Description: <%= food.description %> |
+      Brand Owner: <%= food.brand_owner %> |
+      Ingredients: <%= food.ingredients %>
+    <% end %>
+  </li>
```

Comment:
I would probably add some classes to these elements so that you can utilize `within` blocks to target specific elements/areas and count that the number of elements on the page is indeed correct.